extension/tamper.css

```diff
@@ -31,6 +31,7 @@ h1 {
 label {
   display: block;
   padding: 5px 0;
+  color: blue;
```

Comment:
No worries, enjoy your vacation!

It's a great suggestion, I was not familiar with mix-blend-mode property. I'm testing out the change, but can't seem to get it working as is. I will play with it a bit more... stay tuned.